### DISCUSSION TITLE
can't able to use hooks inside App.js file

### DISCUSSION THREAD

**Discussion by Sriharsh1103** [WalletConnect/walletconnect-monorepo#2982] [Developer Support]
![image_2023_07_04T11_47_54_065Z](https://github.com/WalletConnect/walletconnect-monorepo/assets/133873781/b7eb91d2-ae96-4d75-9717-02a0074ef017)

Hi, if we use :
function App() {
  const status = useAccount();
  
  It's throwing this error
  as shown in screenshot

**Comment by boidushya** [comment]
Wrapping your App component/ Routes with [WagmiConfig](https://wagmi.sh/react/getting-started#wrap-app-with-wagmiconfig) should fix this
